src/Shared/ServerInfrastructure/HttpCharacters.generated.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+
+namespace Microsoft.AspNetCore.Http;
+
+[CompilerGenerated]
```

Comment:
See comment above -- right now this is just copy & paste (+ little editing) from my playgroud-project.